src/martial.py

```diff
@@ -36,11 +36,19 @@ def init_troop(cls, name, nation):
 
         armor = random.choice(nation.basic_armor_list)
 
+        mount = nation.mount_none
+
+        if random.randint(0, 100) < elite + 5:
+            mount = random.choice(nation.basic_mount_list)
+        
+
         tier = 1
 
-        return cls(name, strength, health, 0, ranged, speed, discipline, rank_size, ranks, weapons, armor, elite, tier, [])
+        print "Unknown nation has created a new unit: " + str(name) + ", armed with: " + str(weapons[0].name) +", " + str(weapons[1].name) + ", "+str(armor.name)+", "+str(mount.name)+" elite: "+str(elite)
```

Comment:
Please comment out this print/make an event for this. We should be able to know the nation as we pass it as a parameter to this function.